setup.py

```diff
@@ -304,6 +304,7 @@ def do_setup():
             'configparser>=3.5.0, <3.6.0',
             'croniter>=0.3.17, <0.4',
             'dill>=0.2.2, <0.3',
+            'enum34~=1.1.6',
```

Comment:
n00b qq: what does ~= mean?

Reply:
got it(https://www.python.org/dev/peps/pep-0440/): ```~=: Compatible release clause```

Reply:
It means it will select the latest version of the package, greater or equal to 0.6.10, but still in the 0.6.* version, so it won't download 0.7.0 for example. It ensures you will get security fixes but keep backward-compatibility, if the package maintainer respects the semantic versioning (which states that breaking changes should occur only in major versions).

Or, as said by PEP 440:

> For a given release identifier V.N , the compatible release clause is approximately equivalent to the pair of comparison clauses:
> 
> >= V.N, == V.*